Salary App/Features/SalaryForm/Model/Calculators/StudentLoanCalculator.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  StudentLoanCalculator.swift
+//  Salary App
+//
+//  Created by Tobi Adegoroye on 12/07/2023.
+//
+
+import Foundation
+
+
+struct StudentLoanCalculator {
+    
+    func calculate(salary: Double) -> Double {
+        let rate = 0.09
```

Comment:
There are two different student loan rates, here you may to research this cine it's not always 0.09. Take this into consideration and refactor this function